addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -1195,7 +1198,7 @@ ListRenderer.include({
                             var correspondingRow = self._getRow(recordId);
                             correspondingRow.children().eq(cellIndex).focus();
                         } else if (self.currentGroupId) {
-                                self.$('a[data-group-id=' + self.currentGroupId + ']').focus();
+                            self.$('a[data-group-id="' + self.currentGroupId + '"]').focus();
```

Comment:
This here is a small detail but causes another type of crash (it looks like jQuery doesn't like unquoted strings). This issue seems to only occur on user input and I couldn't reproduce it with tests. Should I look deeper into it ?

Reply:
no it's no big deal